### ISSUE TITLE
Karma for being Annoying issue

### ISSUE THREAD

**Issue by Canatopher** [Chrigi2486/CTModsLC#2]
Tysm for adding the remote to the mod along with a config, but there is one problem - unfortunately this mod does not seem to work on players that don't also have the mod, even if I am the server host. If it is possible, could you please try to make it so that the host having this mod enforces it onto other players in the lobby (even if they dont have it) to properly prevent them from being annoying?
Thanks!

**Comment by Toemmsen96** [comment]
We will look into it, will probably add the possibility to spawn mines via cfg if you are the server host. Spawning explosions without that isnt really possible.

**Comment by Canatopher** [comment]
Thank you for looking into it! Personally, the explosion isnt a huge concern for me, as long as it kills them and only them. If it does spawn a mine under them, there is a bit of concern that it could kill nearby players too that dont deserve it, but thank you for looking into it!

**Comment by Chrigi2486** [comment]
I agree that spawning a mine may introduce some collateral damage and the possibility of griefing.  
We'll look into moving everything to the host (probability calculation and elimination) and having the client only spawning the explosion, as in the end the explosion is just for comedic effect.  
Hence making sure the core functionality of the mod works even if only the host has it installed.  
We'll see if we can implement it when we're back from our holidays.  
In the meantime have a merry Christmas!
